windows/win_lineinfile.py

```diff
@@ -100,20 +100,43 @@
 """
 
 EXAMPLES = """
```

Comment:
I marked examples section as containing regexp. Hopefully that will fix the error.